galaxy_importer/ansible_test/runners/__init__.py

```diff
@@ -31,10 +31,11 @@ def get_runner(cfg):
     if not cfg.run_ansible_test:
         return None
 
-    if not cfg.infra_pulp:
+    if not cfg.ansible_test_local_image:
```

Comment:
Since this value defaults to false, if it is not in the config it will win, I think we want this:
```
    if cfg.infra_osd:
        return OpenshiftJobTestRunner

    if cfg.ansible_test_local_image:
        return LocalImageTestRunner

    return LocalAnsibleTestRunner
```